src/tags/object/AudioNext/view.tsx

```diff
@@ -14,9 +14,9 @@ const AudioNextView: FC<AudioNextProps> = ({ item }) => {
   const [position, setPosition] = useState(1);
   const [audioLength, setAudioLength] = useState(0);
 
-  const [zoom, setZoom] = useState(WS_ZOOM_X.default);
-  const [volume, setVolume] = useState(1);
-  const [speed, setSpeed] = useState(1);
+  const [zoom, setZoom] = useState(Number(item.defaultzoom ?? WS_ZOOM_X.default));
```

Comment:
I think what @hlomzik was referring to with default cases here is that it is redundant due to the definitions in Mobx State Tree already containing optional type and the same default. By the time the code gets to here, it already has the fallback handling taken care of.

Can reduce these checks to:

```suggestion
  const [zoom, setZoom] = useState(Number(item.defaultzoom));
```